Makefile

```diff
@@ -4,11 +4,13 @@ test:
 	cram test
 
 install:
-	install -D -m755 bin/ghu $(DESTDIR)$(PREFIX)/bin/ghu
-	install -D -m755 share/ghu/parse-header $(DESTDIR)$(PREFIX)/share/ghu/parse-header
+	mkdir -p $(DESTDIR)$(PREFIX)/bin
+	mkdir -p $(DESTDIR)$(PREFIX)/share/ghu
+	install -m755 bin/ghu $(DESTDIR)$(PREFIX)/bin/ghu
+	install -m755 share/ghu/parse-header $(DESTDIR)$(PREFIX)/share/ghu/parse-header
 
 uninstall:
 	$(RM) $(DESTDIR)$(PREFIX)/bin/ghu \
-	  $(DESTDIR)$(PREFIX)/share/ghu/parse-header
+		$(DESTDIR)$(PREFIX)/share/ghu/parse-header
```

Comment:
Is this also a portability thing? My understanding is it's not necessary and I don't like.